.github/workflows/rerun-failed-run.yml

```
@@ -12,10 +12,22 @@ on:
       repository:
         description: GitHub repository (ORG/REPO_NAME)
         required: true
+      notify-back-on-error:
```

for consistency with the other inputs above (e.g. `error_messages`, `run_id`):

```suggestion
      notify_back_on_error:
```